IPs/RSKIP272.md

```diff
@@ -0,0 +1,102 @@
+# Bridge UTXO Management Account
+
+
+|RSKIP          | 272 |
+| :------------ |:-------------|
+|**Title**      |Bridge UTXO Management Account|
+|**Created**    |AUG-2021 |
+|**Author**     | SDL |
+|**Purpose**    |Sec, Usa |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Draft |
+
+#  **Abstract**
+
+One of the problems of the RSK bridge is that peg-out fees are unknown to the user until the peg-out is executed, and the fees are decremented from the peg-out amount. The user has no control of the final amount to be received. This RSKIP proposes a new algorithm for the Bridge to pay for all peg-outs, and charge the users a predictable dynamically-adjusted fee. The results is a much better user experience.
+
+## Motivation
+
+The current RSK bridge can suffer from a number of problems described in [RSKIP265](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP265.md). In this RSKIP we're mostly concerned with the variable peg-out costs. The proposed solution is to provide to the user a method to query the fees to be paid, and charge that amount to the user. This is done by maintaining a separate Bridge balance used for peg-out fee payments. In the future this balance can be used for other UTXO management activities, such as refresh of UTXO time-locks. 
+
+## Specification
+
+The Bridge contract will try to maintain a **UTXO Maintenance Account (UMA)** with a balance high enough to pay for peg-outs, UTXO consolidations (if [RSKIP171](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP271.md) is activated), time-lock refresh (if [RSKIP264](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP264.md) is activated) and other maintenance operations without the need to charge peg-in or peg-out users. 
+
+The bridge computes two values: Fi and Fo. Fi represents the cost in fees of consuming a single Powpeg input, and Fo is the cost of one P2SH output.
+
+We define a threshold T(n) as the fee cost of a transactions consuming n Powpeg Bitcoin inputs. We compute T(n) as (n\*Fi+2\*Fo).
+If the UMA balance is lower than T(C), then peg-outs users will be charged 10% more, and the Bridge will move the collected extra fee to the UMA account. If the balance is higher than T(B), then the peg-outs will be charged 10% less. The values C and B are defined in the following way:
+
+If peg-out batching is activated ([RSKIP271](https://github.com/rsksmart/RSKIPs/blob/master/IPs/RSKIP271.md)) then the constants selected are B=60 and C=30.
+
+If peg-out batching is not activated, then B=600 and C=300.
```

Comment:
These 10X numbers (300/600) instead of (30/60) ... are they referenced anywhere else? I don't recall them in related IPs .. maybe I missed them

Reply:
I added a paragraph in the rationale explaining them